frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pages/citizen/Create/index.js

```
@@ -1,178 +1,800 @@
+
+  const { data: commonFields, isLoading } = Digit.Hooks.pt.useMDMS(Digit.ULBService.getStateId(), "PropertyTax", "CommonFieldsConfig");
+  console.log("commonFieldsjjj", commonFields)
+  const token = localStorage.getItem("token");
+  const assessmentYears = [
+    { code: "2024-25", name: "2024-25" },
+    { code: "2023-24", name: "2023-24" },
+    { code: "2022-23", name: "2022-23" }
+  ];
+let tenantIdss = Digit.ULBService.getCurrentTenantId();
+console.log(tenantIdss,"tenantIdss")
+  let userInfo1 = JSON.parse(localStorage.getItem("user-info"));
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'userInfo1' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)